src/Server.cpp

```diff
@@ -270,7 +270,15 @@ int Server<IoType, SwitchActivityInfo, SwitchCalcGaps>::server_accept(int ifd)
 		tmp->recv.cur_offset = 0;
 		tmp->recv.cur_size = tmp->recv.max_size;
 
+#ifdef  USING_VMA_EXTRA_API
+		if (g_pApp->m_const_params.fd_handler_type == VMAPOLL && g_vma_api){
+			active_ifd = g_vma_comps->user_data;
```

Comment:
Please consider checking `g_vma_api` as first in all places because other stuff can not be workable if `g_vma_api == NULL`.